resources/settings.xml

```diff
@@ -10,8 +10,10 @@
         <setting label="30904" type="text" id="warning" enable="false"/> <!-- disabled_warning -->
         <setting label="30905" help="30906" type="slider" id="update_frequency" default="31" range="1,3,90" option="int" enable="eq(-2,false)" visible="!system.platform.android"/>
         <setting label="30907" help="30908" type="folder" id="temp_path" source="" option="writeable" default="special://masterprofile/addon_data/script.module.inputstreamhelper" visible="!system.platform.android"/>
+        <setting label="30913" help="30914" type="slider" id="backups" default="4" range="0,1,5" option="int" visible="!system.platform.android"/>
```

Comment:
I would not limit this to 5. I would say, make it go up to 15 or 20.
This would be something useful for us in development too.

Reply:
#165, second line...
But since I thought these were a bit few, too, I'll increase it.

Reply:
@horstle Sorry for changing my mind during review. Back then I was not thinking of a slider with a limitation. (I do prefer the slider though)